test/invoker.ts

```diff
@@ -156,38 +156,51 @@ describe('CloudEvents request to event function', () => {
     headers: { [key: string]: string };
     body: {};
   }
+
+  const specversion = '1.0';
+  const type = 'com.google.cloud.storage';
+  const source =
+    'https://github.com/GoogleCloudPlatform/functions-framework-nodejs';
+  const subject = 'test-subject';
+  const id = 'test-1234-1234';
+  const time = '2020-05-13T01:23:45Z';
+  const dataschema =
```

Comment:
This isn't a true dataschema of the payload. We should just not include this CE attr.

Reply:
Removed!